tools/mountsnoop.py

```diff
@@ -416,14 +433,19 @@ def main():
         help=argparse.SUPPRESS)
     parser.add_argument("-P", "--parent_process", action="store_true",
         help="also snoop the parent process")
+    parser.add_argument("--cgroupmap",
+        help="trace cgroups in this BPF map only")
+    parser.add_argument("--mntnsmap",
+        help="trace mount namespaces in this BPF map only")
     args = parser.parse_args()
 
     mounts = {}
     umounts = {}
+    local_bpf_text = filter_by_containers(args) + bpf_text
     if args.ebpf:
-        print(bpf_text)
+        print(local_bpf_text)
         exit()
-    b = bcc.BPF(text=bpf_text)
+    b = bcc.BPF(text=local_bpf_text)
```

Comment:
I don't think you need an additional variable. 

```    
bpf_text = filter_by_containers(args) + bpf_text
```
and use `bpf_text` everywhere else should work just fine.

Reply:
Thank you for the review.

I wanted to address it this evening, but I deleted minikube and my connection is quite unstable, so it is a bit hard to finish the `go mod download`...
I will try to correct it before Sunday.

Reply:
I tested the code you proposed and I remembered why I use the `local` thing because I got this error:
```
Traceback (most recent call last):
  File "/usr/share/bcc/tools/mountsnoop", line 474, in <module>
    main()
  File "/usr/share/bcc/tools/mountsnoop", line 444, in main
    bpf_text = filter_by_containers(args) + bpf_text
UnboundLocalError: local variable 'bpf_text' referenced before assignment
```

Indeed, since this code `bpf_text = filter_by_containers(args) + bpf_text` is in `main()`, python allocates a local variable called `bpf_text` which is used uninitialized, so I got an error.
So, using a local variable called `local_bpf_text` avoids this problem.
Finally, I solved it using:
```python
global bpf_text
bpf_text = filter_by_containers(args) + bpf_text
```
which seems to be a more pythonic way than having `local_bpf_text`.